ctags.lua

```diff
@@ -220,12 +220,32 @@ local function tselect_cmd(tag, force)
 		vis:info(string.format('Tag not found: %s', tag))
 	else
 		local keys = {}
-		for i = 1, #matches do
-			table.insert(keys, matches[i].desc)
+
+		-- find max width for pretty printing
+		local maxdescwidth = 0
+		for _, match in ipairs(matches) do
+			if match.desc:len() > maxdescwidth then
+				maxdescwidth = match.desc:len()
+			end
+		end
+		local fmt = '%-'..maxdescwidth..'s %s'
```

Comment:
There is an edge case: `maxdescwidth` cannot be longer than 99 characters (`fmt:format` limitation). Maybe limit to something like 40? We would also need to take the last N characters to make sure that the filename is visible for longer strings.